communitiesuk/funding-service-design-assessment
# Added num2words module and ordinals go higher than 5

FS-3168

### Change description
Fixed bug where raised flag after 5 doesn't show the ordinal words. For example "First" to "Fifth" would show, now it it can show any number


### How to test
Add more than 5 flags and check the flag banners to see if the wording is correct.

### Screenshots of UI changes (if applicable)
![image](https://github.com/communitiesuk/funding-service-design-assessment/assets/60141291/ee0809fd-ab1d-4e37-a47f-fc660325aab8)
